config/config.env

```diff
@@ -0,0 +1,93 @@
+S3_GW_ADDRESS=NfgHwwTi3wHAS8aFAN243C5vGbkYDpqLHP
+# Passphrase to decrypt wallet.
+S3_GW_WALLET_PASSPHRASE=s3
+
+# Nodes
+# This configuration makes the gateway use the first node (grpc://s01.neofs.devenv:8080)
+# while it's healthy. Otherwise, gateway uses the second node (grpc://s01.neofs.devenv:8080)
+# for 10% of requests and the third node (grpc://s03.neofs.devenv:8080) for 90% of requests.
+# Until nodes with the same priority level are healthy
+# nodes with other priority are not used.
+# The lower the value, the higher the priority.
+S3_GW_PEERS_0_ADDRESS=grpc://s01.neofs.devenv:8080
+S3_GW_PEERS_0_PRIORITY=1
+S3_GW_PEERS_0_WEIGHT=1
+S3_GW_PEERS_1_ADDRESS=grpc://s02.neofs.devenv:8080
+S3_GW_PEERS_1_PRIORITY=2
+S3_GW_PEERS_1_WEIGHT=0.1
+S3_GW_PEERS_2_ADDRESS=grpc://s03.neofs.devenv:8080
+S3_GW_PEERS_2_PRIORITY=2
+S3_GW_PEERS_2_WEIGHT=0.9
+
+# Address to listen and TLS
+S3_GW_LISTEN_ADDRESS=0.0.0.0:8080
+S3_GW_TLS_CERT_FILE=/path/to/tls/cert
+S3_GW_TLS_KEY_FILE=/path/to/tls/key
+
+# Config file
+S3_GW_CONFIG=/path/to/config/yaml
+
+# Logger
+S3_GW_LOGGER_LEVEL=debug
+
+# RPC endpoint and order of resolving of bucket names
+S3_GW_RPC_ENDPOINT=http://morph-chain.neofs.devenv:30333/
+S3_GW_RESOLVE_ORDER="nns dns"
+
+# Metrics
+S3_GW_METRICS=false
+S3_GW_PPROF=false
+
+# Timeout to connect to a node
+S3_GW_CONNECT_TIMEOUT=30s
+# Timeout to check node health during rebalance.
+S3_GW_HEALTHCHECK_TIMEOUT=15s
+# Interval to check node health
+S3_GW_REBALANCE_INTERVAL=15s
+
+# Limits for processing of clients' requests
+S3_GW_MAX_CLIENTS_COUNT=100
+# Deadline after which the gate sends error `RequestTimeout` to a client
+S3_GW_MAX_CLIENTS_DEADLINE=30s
+
+# Caching
+# Cache for objects
+S3_GW_CACHE_OBJECTS_LIFETIME=5m
+S3_GW_CACHE_OBJECTS_SIZE=1000000
+# Cache which keeps lists of objects in buckets
+S3_GW_CACHE_LIST_LIFETIME=1m
+S3_GW_CACHE_LIST_SIZE=100000
+# Cache which contains mapping of bucket name to bucket info
+S3_GW_CACHE_BUCKETS_LIFETIME=1m
+S3_GW_CACHE_BUCKETS_SIZE=1000
+# Cache which contains mapping of nice name to object addresses
+S3_GW_CACHE_NAMES_LIFETIME=1m
+S3_GW_CACHE_NAMES_SIZE=10000
+ # Cache for system objects in a bucket: bucket settings, notification configuration etc
+S3_GW_CACHE_SYSTEM_LIFETIME=5m
+S3_GW_CACHE_SYSTEM_SIZE=100000
+# Cache which stores access box with tokens by its address
+S3_GW_CACHE_ACCESSBOX_LIFETIME=10m
+S3_GW_CACHE_ACCESSBOX_SIZE=100
+
+# NATS
+S3_GW_NATS_ENABLED=true
+S3_GW_NATS_ENDPOINT=nats://nats.neofs.devenv:4222
+S3_GW_NATS_TIMEOUT=30s
+S3_GW_NATS_CERT_FILE=/path/to/cert
+S3_GW_NATS_KEY_FILE=/path/to/key
+S3_GW_NATS_ROOT_CA=/path/to/ca
+
+# Default policy of placing containers in NeoFS
+# If a user sends a request `CreateBucket` and doesn't define policy for placing of a container in NeoFS, the S3 Gateway
+# will put the container with default policy. It can be specified via environment variable, e.g.:
+S3_GW_DEFAULT_POLICY="REP 1 CBF 1 SELECT 1 FROM *"
```

Comment:
Why is this setting different from the one in `config.yaml`?

Reply:
It's just another example of possible container policy, I didn't suppose it must be a default value or the same as another config. But okay, replaced.